doc/tut1.rst

```diff
@@ -1441,31 +1441,111 @@ string that is "useless" and replace it with "useful".
 Note: alternate ways of writing this are ``b[^8..^2] = "useful"`` or
 as ``b[11..b.len-2] = "useful"`` or as ``b[11..<b.len-1] = "useful"``.
 
+Objects
+-------
+
+The default type to pack different values together in a single
+structure with a name is the object type. An object is a value type,
+which means that when an object is assigned to a new variable all its
+components are copied as well.
+
+Each object type ``T`` has a constructor ``T(field: value, ...)``
+where all of its fields can be initialized.  Unspecified fields will
+get their default value.
```

Comment:
"...will get their default value *if the type has one*", e.g.

```nim
type
  Person = object
    name: string
    age: int
    fingers: range[1 .. 5]
```

You cannot do `let person4 = Person(age: 3)` here.

Reply:
true. I can change it. I just wonder if I should go into this detail here. It is after all a tutorial and I think we can put the reader in a simplified view of the world.

Reply:
I decided to not change this. Despite the fact that you are right. I think this detail information is just a nitpick that doesn't add any value to understand the language. In your example the error message is also pretty straight to the point.